packages/sdk/src/swap/v2/types.ts

```diff
@@ -83,13 +83,16 @@ interface EgressFields {
   sentAtBlockIndex: string | undefined;
   ignoredAmount: string | undefined;
   failure: Failure | undefined;
+  failedAt: number | undefined;
+  failedAtBlockIndex: string | undefined;
 }
 interface SwapStatusResponseCommonFields extends ChainsAndAssets {
   swapId: string;
   destAddress: string;
   ccm: Ccm | undefined;
   boost: Boost | undefined;
   estimatedDurationSeconds: number | null | undefined;
+  srcChainRequiredBlockConfirmations: number | null;
```

Comment:
moved this here because it's true for vault swaps too (even if we don't currently create a swap record until it's confirmed)